### PR TITLE
Add usage with Tunnelmole to docs

### PR DESCRIPTION
**Pull request by robbie-cahill**
This PR adds [Tunnelmole](https://github.com/robbie-cahill/tunnelmole-client) as a tunneling option.

Tunnelmole is a FOSS tunneling solution with a growing community. It works out of the box and can be optionally self hosted with the [Tunnelmole Service](https://github.com/robbie-cahill/tunnelmole-service). Both the client and service are open source.

Example:
```
➜  ~ tmole 8000
http://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
https://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
```

### CODE REVIEW

File: docs/docs/30-administration/70-proxy.md [hunk 1]
```diff
@@ -98,13 +98,40 @@ woodpeckeragent.example.com {
...
+
 ## Ngrok
 
-After installing [ngrok](https://ngrok.com/), open a new console and run:
+[Ngrok](https://ngrok.com/) is a popular closed source tunnelling tool. After installing ngrok, open a new console and run the following command:
 
 ```bash
 ngrok http 8000
-```
+````
```
Comment (micash545) [inline docs/docs/30-administration/70-proxy.md:134]: Code block are closed by 3 `

Comment (robbie-cahill) [reply docs/docs/30-administration/70-proxy.md:131]: Looks like I missed that little typo, this is now fixed.

File: docs/docs/30-administration/70-proxy.md [hunk 2]
```diff
@@ -98,13 +98,40 @@ woodpeckeragent.example.com {
...
 :::
 
+## Tunnelmole
+
+[Tunnelmole](https://github.com/robbie-cahill/tunnelmole-client) is an open source tunneling tool. Install Tunnelmole with one of the following methods depending on your system:
+
+- NPM: `npm install -g tunnelmole`
+- Linux: `curl -s https://tunnelmole.com/sh/install-linux.sh | sudo bash`
+- Mac: `curl -s https://tunnelmole.com/sh/install-mac.sh --output install-mac.sh && sudo bash install-mac.sh`
+- Windows: Install with NPM, or if you don't have NodeJS installed, download the `exe` file for Windows [here](https://tunnelmole.com/downloads/tmole.exe) and put it somewhere in your PATH.
```
[the remark this answers is not in the document]
Comment (robbie-cahill) [reply docs/docs/30-administration/70-proxy.md]: Done, linked to the installation instructions in the GitHub readme file.

File: docs/docs/30-administration/70-proxy.md [hunk 3]
```diff
@@ -98,13 +98,37 @@ woodpeckeragent.example.com {
 Above configuration shows how to create reverse-proxies for web and agent communication. If your agent uses SSL do not forget to enable [WOODPECKER_GRPC_SECURE](./15-agent-config.md#woodpecker_grpc_secure).
 :::
 
+## Tunnelmole
+
+[Tunnelmole](https://github.com/robbie-cahill/tunnelmole-client) is an open source tunneling tool.
+
+First, [Install tunnelmole](https://github.com/robbie-cahill/tunnelmole-client#installation).
```
Comment (anbraten) [inline docs/docs/30-administration/70-proxy.md:105]:
```suggestion
Start by [installing tunnelmole](https://github.com/robbie-cahill/tunnelmole-client#installation).
```

File: docs/docs/30-administration/70-proxy.md [hunk 4]
```diff
@@ -98,13 +98,37 @@ woodpeckeragent.example.com {
...
+
+After installation, run the following command:
```
Comment (anbraten) [inline docs/docs/30-administration/70-proxy.md:107]:
```suggestion
After the installation, run the following command to start tunnelmole:
```

**Review by anbraten — APPROVED** [review]

File: docs/docs/30-administration/70-proxy.md [hunk 5]
```diff
@@ -98,13 +98,37 @@ woodpeckeragent.example.com {
...
+
+```bash
+➜  ~ tmole 8000
+http://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
+https://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
+```
+
+Set `WOODPECKER_HOST` (for example in `docker-compose.yml`) to the Tunnelmole URL (`xxx.tunnelmole.net`) and start the server.
+
+````
```
Comment (anbraten) [inline docs/docs/30-administration/70-proxy.md:123]:
```suggestion
```

File: docs/docs/30-administration/70-proxy.md [hunk 6]
```diff
@@ -98,13 +98,35 @@ woodpeckeragent.example.com {
...
+
 ## Ngrok
 
-After installing [ngrok](https://ngrok.com/), open a new console and run:
+[Ngrok](https://ngrok.com/) is a popular closed source tunnelling tool. After installing ngrok, open a new console and run the following command:
 
 ```bash
 ngrok http 8000
-```
+````
```
Comment (anbraten) [inline docs/docs/30-administration/70-proxy.md:129]:
```suggestion
```
```

**Pull request merged**